build.properties

```diff
@@ -41,6 +41,7 @@
 # are contained in the JMeter binary release.
 
 maven2.repo                 = https://repo1.maven.org/maven2
+maven.repo                  = https://repo.maven.apache.org/maven2
```

Comment:
AFAIK, LatencyUtils is available in Maven Central. Why do you add `maven.repo`?

Reply:
agreed